instrumentation/process/context.go

```diff
@@ -44,7 +44,14 @@ func SpanContext() opentracing.SpanContext {
 	return processSpanContext
 }
 
-func StartSpan(operationName string, opts ...opentracing.StartSpanOption) opentracing.Span {
+func StartSpan(opts ...opentracing.StartSpanOption) opentracing.Span {
+	if spanCtx := SpanContext(); spanCtx != nil {
+		opts = append(opts, opentracing.ChildOf(spanCtx))
+	}
+	return instrumentation.Tracer().StartSpan(getOperationNameFromArgs(os.Args), opts...)
+}
+
+func StartSpanWithOperationName(operationName string, opts ...opentracing.StartSpanOption) opentracing.Span {
```

Comment:
I'd not offer this method. If the user wants to start span with an operational name or other options, they can do it manually using opentracing directly IMHO.

Reply:
Ok for me, I've rename the old method, removing it then...

Reply:
`StartSpanFromContext` has a dependency with this method, so I'm gonna change it to private.